pages/settings/wallets/SetupWallet.tsx

```diff
@@ -119,7 +119,10 @@ export function SetupWallet() {
       position: "top",
     });
 
-    router.replace("/");
+    if (router.canDismiss()) {
+      router.dismissAll();
+    }
+    router.navigate("/");
```

Comment:
is this navigation even needed if we dismiss all the routes? shouldn't this be the default?

Reply:
Won't this also push a route onto the stack, leaving a back button?